client/signup/main.jsx

```diff
@@ -124,11 +123,6 @@ const Signup = React.createClass( {
 		} );
 		recordSignupStart();
 
-		// Signup updates the cart through `SignupCart`. To prevent
-		// synchronization issues and unnecessary polling, the cart is disabled
-		// here.
-		disableCart();
```

Comment:
Thats a complicated question really, I'm not sure what's the answer to this one.

From a certain point of view you _are_ right, we should clear the cart, from another point of view no, we should do something else if user has items in his cart, like, maybe, ask him if he want to check-out using this items?

I'm not sure what's the answer here.

BTW, that is also true for the previous approach with `disableCart()`, disabling the cart isn't clearing the cart, just disabling the fetch polling, but if the cart is full from a prev. "stale data" as far as I understand.